windows-driver-docs-pr/stream/360-camera-video-capture.md

```diff
@@ -114,9 +114,9 @@ AddReg=ContosoSampleDeviceMFT_COM.AddReg
 ;-----------------------------------------------------------------------------------
 
 [ContosoSampleDeviceMFT_COM.AddReg]
-HKCR,CLSID\%SampleDeviceMFT.CLSID%,,,%SampleDeviceMFT.FriendlyName%
-HKCR,CLSID\%SampleDeviceMFT.CLSID%InProcServer32\,,,%%SystemRoot%%\ContosoSampleDeviceMFT.dll
-HKCR,CLSID\%SampleDeviceMFT.CLSID%InProcServer32\,ThreadingModel,,"Both"
+HKR,Classes\CLSID\%SampleDeviceMFT.CLSID%,,,%SampleDeviceMFT.FriendlyName%
```

Comment:
Above you target build 22000, but isn't the HKR COM registration only supported by the camera stack in a much later build than that?